libs/metrics/base.py

```diff
@@ -204,9 +204,9 @@ def compute_live_metrics(cls, config, live_interval):
             report_file_exists = os.path.isfile(live_report_csv)
             if not os.path.isfile(today_entity_csv):
                 return
-            entity["base_directory"] = entity_directory
             entries_in_interval = int(live_interval * 60 / get_source_logging_interval(config))
             live_data = cls.generate_live_csv_data(config, today_entity_csv, entity, entries_in_interval)
+            assert len(live_data) == len(csv_headers), "Row element count not the same as header count!!"
```

Comment:
Why is this here? To indicate logs should be deleted?

Reply:
I found a bug where we were adding too more items to the `live_data` than headers. As we manually take the first N items from `live_data` where N = len(headers) there would be no exception, but the final data was wrong. This is to avoid such errors in future